msmbuilder/cluster/agglomerative.py

```diff
@@ -142,6 +142,8 @@ class _LandmarkAgglomerative(ClusterMixin, TransformerMixin):
     ----------
     landmark_labels_
     landmarks_
+    cluster_centers_ : mean of all points in each cluster along every dimension,
```

Comment:
I don't understand your question- can you rephrase? For each cluster, I am taking the average along each dimension and reporting the 'center' as the point defined by those averages.